install_config/topics/configuring_gce_objects.adoc

```diff
@@ -0,0 +1,176 @@
+entered.
+
+See link:https://cloud.google.com/billing/docs/how-to/manage-billing-account[Create, Modify, or Close Your Billing Account in the GCP documentation] for more information.
+
+Cloud identity and access management::
+
+Deploying {product-title} requires the proper permissions. A user must be able
+to create service accounts, cloud storage, instances, images, templates, Cloud
+DNS entries, and deploy load balancers and health checks. Delete permissions are
+also helpful in order to be able to redeploy the environment while testing.
+
+You can create service accounts with specific permissions, then use them to
+deploy infrastructure components instead of regular users. You can also create
+roles to limit access to different users or service accounts.
+
+GCP instances use service accounts to allow applications to call GCP APIs. For
+example, {product-title} node hosts can call the GCP disk API to provide a
+persistent volume to an application.
+
+Access control to the various infrastructure, service resources, and
+fine-grained roles are available using the IAM service. For more information, see link:https://cloud.google.com/compute/docs/access/[the Access cloud overview section of the GCP documentation].
+
+SSH keys::
+
+GCP injects SSH public keys as authorized keys to be able to log in using SSH in
+the created instances. The list of SSH keys can be configured per instance or
+per project.
+
+You can use existing SSH keys. GCP metadata can help store the SSH keys that are
+injected at boot time in the instances to allow SSH access.
+
+See link:https://cloud.google.com/compute/docs/storing-retrieving-metadata[the Metadata section of the GCP documentation] for more information. 
+
+GCP regions and zones::
+
+GCP has a global infrastructure that covers regions and availability
+zones. While deploying {product-title} in GCP on different zones can help avoid
+single-point-of-failures, there are some caveats regarding storage. 
+
+GCP disks are created within a zone. Therefore, if a {product-title} node host
+goes down in zone "A" and the pods should be moved to zone "B", the persistent
+storage cannot be attached to those pods because the disks are in a different
+zone.
+
+Deploying a single zone of multizone {product-title} environment is an important
+decision to make before installing {product-title}. If deploying a multizone
+environment, the recommended setup is to use three different zones in a single region.
+
+See link:https://cloud.google.com/compute/docs/zones[the GCP documentation on regions and zones] or link:https://cloud.google.com/kubernetes-engine/docs/concepts/regional-clusters[the Kubernetes documentation on multiple zones] for more information.
+
+External IP address::
+
+So that GCP instances can communicate with the Internet, you must attach an
+external IP address to the instance. Also, an external IP address is required to
+communicate with instances deployed in GCP from outside the VPC Network.
```

Comment:
I'd spell out VPC on first use unless you're sure that I'm the only person who should know what it stands for and doesn't. :wink: